hydrus/parser/openapi-hydra_parser.py

```diff
@@ -0,0 +1,121 @@
+"""
+Module to take in Open Api Specification and convert it to HYDRA Api Doc
+
+"""
+import yaml
+import json
+
+from hydrus.hydraspec.doc_writer import HydraDoc, HydraClass, HydraClassProp, HydraClassOp
+
+with open("../samples/openapi_sample.yaml", 'r') as stream:
+    try:
+        doc = yaml.load(stream)
+    except yaml.YAMLError as exc:
+        print(exc)
+
+
+definitionSet = set()
+classAndClassDefinition = dict()
+hydra_doc = ""
+
+
+def getClasses():
+    """
+    Get Definitions from Open Api specification and convert to Classes and supported Props for Hydra Documentation
+    """
+    definitions = doc["definitions"]
+
+    for class_ in definitions:
+        try:
+            desc = definitions[class_]["description"]
+        except KeyError:
+            desc = class_
+        classDefinition = HydraClass(class_, class_, desc, endpoint=False)
+        properties = definitions[class_]["properties"]
+        classAndClassDefinition[class_] = classDefinition
+        definitionSet.add(class_)
+        api_doc.add_supported_class(classDefinition, collection=False)
+        for prop in properties:
+            new_prop = HydraClassProp("vocab:" + prop, prop, required=False, read=True, write=True)
```

Comment:
maintaining  line length < 80 according to PEP8, will be better.

Reply:
okay